instrument/IN5_Definition.xml

```diff
@@ -41,664 +42,664 @@
   <idlist idname="monitors">
     <id val="100000"/>
   </idlist>
-  <type is="detector" name="standard_pixel">
+  <type name="standard_pixel" is="detector">
     <cylinder id="pixel_shape">
-      <centre-of-bottom-base x="0.0" y="-0.00612033195021" z="0.0"/>
+      <centre-of-bottom-base x="0.0" y="-0.006120331950207469" z="0.0"/>
       <axis x="0.0" y="1.0" z="0.0"/>
-      <radius val="0.0122"/>
-      <height val="0.0122406639004"/>
+      <radius val="0.012199999999999999"/>
```

Comment:
I do wonder why there is so many changes to the IDFs of IN5, PANTHER, and SHARP, and most of them seem to be minimal/stylistic. Where do they stem from? Also, are they taken in the account in the IDF generation script on the mantidgeometry repository?

Reply:
If I recall, adding the flip parameter to the generator (not sure if I pushed that to mantid-geometry, I need to check) meant I needed to rerun the scripts and for some reason all positions got moved by a absurdly tiny distance and so the diff is huge without any relevant change